app/helpers/campaign_helper.rb

```diff
@@ -15,6 +15,6 @@ def nav_link(link_text, link_path)
   def html_from_markdown(markdown)
     return unless markdown
     converter = Redcarpet::Markdown.new(Redcarpet::Render::HTML)
-    raw converter.render(markdown)
+    sanitize converter.render(markdown)
```

Comment:
I'm not sure what the difference here is or whether this could cause bugs. This change shouldn't be part of this PR.

Reply:
Okay, I've removed it